jobs/candlepinPerformanceTestsJob.groovy

```diff
@@ -1,4 +1,9 @@
-job("Candlepin Performance"){
+import jobLib.rhsmLib
+
+String baseFolder = rhsmLib.candlepinJobFolder
+
+job("$baseFolder/Candlepin Performance"){
```

Comment:
pedantic request: inconsistent spacing with curly braces. I would prefer a space before it here.